pkg/init/Dockerfile

```diff
@@ -1,3 +1,9 @@
+FROM alpine:edge AS build
```

Comment:
it does, but it's in a APK mirror. You still need to `apk add` them as in:
```
FROM linuxkit/alpine:630ee558e4869672fae230c78364e367b8ea67a9 AS build
RUN mkdir -p /out/etc/apk && cp -r /etc/apk/* /out/etc/apk/
RUN apk add --no-cache --initdb -p /out alpine-baselayout make gcc
...
```
[note to self that I need to rewrite the packages document...]